sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -421,68 +427,77 @@ export class MessageSender extends LinkEntity {
   }
 
   /**
-   * Initializes the sender session on the connection.
+   * Initializes the `ClientEntityContext.sender` associated with this
+   * `MessageSender`.
+   *
+   * If the connection is already open this method resolves immediately.
```

Comment:
Its not just the AMQP connection, we need the AMQP link to be open as well. I would suggest the below to avoid confusion between the generic term connection and the actual AMQP connection

```suggestion
   * If the underlying AMQP sender link is already open this method resolves immediately.
```